lib/aur-query

```diff
@@ -3,8 +3,13 @@
 [[ -v AUR_DEBUG ]] && set -o xtrace
 argv0=query
 AUR_LOCATION=${AUR_LOCATION:-'https://aur.archlinux.org'}
+AUR_QUERY_PARALLEL=${AUR_QUERY_PARALLEL:-0}
+AUR_QUERY_PARALLEL_MAX=${AUR_QUERY_PARALLEL_MAX:-15}
```

Comment:
Cool! Let's have it parallel by default, and maybe even with 50 threads as curl defaults to?

```sh
AUR_QUERY_PARALLEL=${AUR_QUERY_PARALLEL:-1}
AUR_QUERY_PARALLEL_MAX=${AUR_QUERY_PARALLEL_MAX:-50}
```